### ISSUE TITLE
Column preferences not saved

### ISSUE THREAD

**Issue by dhyanKaro** [losnoco/Cog#396]
**Describe**
Removed column preferences in the playlist display area are not preserved across runs.

**To Reproduce** (delete if not applicable)
Right click on any column.
Deselect one.
Restart

**Version information:***
 - macOS version: 10.14.6
 - Cog version: ... 2859

**Comment by kode54** [comment]
This is a known bug with 10.14.x. I don't know how to fix it, or if it is even fixable.

**Comment by dayfuaim** [comment]
It also "works" vice versa: can't add one or more columns – just nothing happens.

**Comment by kode54** [comment]
Guess I can "fix" this by removing support for 10.14 and older.

**Comment by ThomRoss** [comment]
@kode54 😱 For the love of everything on this ball of mud we call Earth, please don't.

**Comment by kode54** [comment]
> @kode54 😱 For the love of everything on this ball of mud we call Earth, please don't.

@ThomRoss Yeah, nah, I won't be doing that. I may, however, decide to completely implement my own preferences saving, including my own hooks for saving window sizes and positions and column layouts manually. I think I want to use nlohmann json.

Anything to avoid the bullcrap of exactly 10.14.x breaking various System Defaults / Preferences saving and/or loading.

**Comment by mattneub** [comment]
This is perhaps a variant of https://github.com/losnoco/Cog/issues/376. (I am "ghost" in that conversation.)
